tests/gis_tests/layermap/tests.py

```diff
@@ -324,6 +325,20 @@ def allow_relation(self, obj1, obj2, **hints):
     def allow_migrate(self, db, app_label, **hints):
         return True
 
+    def test_null_geom_with_unique(self):
+        """ Test a null geom field, with unique keyword argument """
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things.

Reply:
Going with "Unique with a null column should not throw exception"